Big-Spender/readme.md

```diff
@@ -68,39 +69,48 @@ INSERT YOUR QUERY HERE
 **You:** Then here's the query for that:
 
 ```sql
-INSERT YOUR QUERY HERE
+INSERT : select transaction_no, description from spends where lower(description) like '%fee%';
 ```
 
 **Farnoosh:** Hi, it's me again. It turns out we also need the transactions that have the expense area of 'Better Hospital Food'. Can you help us with that one?
 
 **You:** No worries. Here's the query for that:
 
 ```sql
-INSERT YOUR QUERY HERE
+INSERT: SELECT * from spends join expense_areas on (expense_areas.id= spends.expense_area_id) where expense_area='Better Hospital Food'; ;
+ For transcation nr only  : SELECT transaction_no from spends join expense_areas on (expense_areas.id= spends.expense_area_id) where expense_area='Better Hospital Food'; ;
+
 ```
 
 **Claire:** Great, that's very helpful. How about the total amount spent for each month?
 
 **You:** You can get that by using the GROUP BY clause. Here's the query:
 
 ```sql
-CREATE YOUR QUERY HERE
+CREATE : SELECT EXTRACT(MONTH FROM date) AS month, SUM(amount) AS total_amount
+FROM spends
+GROUP BY EXTRACT(MONTH FROM date);
 ```
 
 **Farnoosh:** Thanks, that's really useful. We also need to know the total amount spent on each supplier. Can you help us with that?
 
 **You:** Sure thing. Here's the query for that:
 
 ```sql
-INSERT YOUR QUERY HERE
+INSERT: SELECT supplier_id, SUM(amount) AS total_amount
+FROM spends
+GROUP BY supplier_id;
 ```
 
 **Farnoosh:** Oh, how do I know who these suppliers are? There's only numbers here.
 
 **You:** Whoops! I gave you ids to key the totals, but let me give you names instead.
 
 ```sql
-INSERT YOUR QUERY HERE
+INSERT :SELECT sup.supplier, SUM(sp.amount) AS total_amount
+FROM spends AS sp
+JOIN suppliers AS sup ON sp.supplier_id = sup.id
+GROUP BY sup.supplier;
```

Comment:
nice job-- what if we wanted to do this by month and supplier? How might you do that ;)